news/2018-07-18-the-qat-gazette-june-2018.md

```diff
@@ -0,0 +1,98 @@
+
+June was a fairly hectic month for the Quality Assurance Team. We had our second ever public QAT meeting and from that, we emerged with a new vision and direction for the team, along with a slight restructuring of how responsibilities are shared.
+
+As a result, we said goodbye to some old and long serving friends, and welcomed aboard a new member to the team. It's quite a lot to keep track of, but we'll head through one thing at a time, so please read on to stay on top of the latest changes and news!
+
+### Public QAT Meeting
+
+On June 17th, we had our second public QAT meeting in the osu!dev server, after the success of the first back in April. This meeting was advertised much more in advance, and this was reflected by a considerably higher turn out with more community involved discussion.
+
+Community members were able to share some of their opinions on a plethora of topics, including changes to the Beatmap Nominators subdivision system, beatmap veto mediation, tighter quality standards and review of our Beatmap Nominators, introduction of QAT interviews, and changes to Beatmap Nominator applications.
+
+Don't worry if you weren't able to make it for the meeting – you can still catch up and get the lowdown using the Beatmap Management Forum [summary article](https://osu.ppy.sh/community/forums/topics/762133).
+
+Given the meeting's success, we naturally plan to host more public discussions in the future, so stay tuned as we’ll make sure to announce them in advance right here in the Gazette before anywhere else. These are hosted in the [public osu!dev discord server](https://discord.gg/ppy), so be sure to join and verify yourself if you haven’t done so already.
+
+### osu!mania Ranking Criteria Draft
+
+After three long months, the new osu!mania mode-specific Ranking Criteria draft is out for review now. We've taken community feedback and incorporated it into the current Ranking Criteria draft, including clarifications and additions that are hopefully less vague and more intuitive to read.
+
+We've also added a few new Rules and converted some old Rules into Guidelines, to be more flexible and afford more freedom to the hard working mappers and modders who bring new content into the Ranked section.
+
+Stop by and give the [current draft](https://gist.github.com/Okorin/1a5f5d6f489494c26b8a4fec13ba69ba) a read, and drop your feedback and opinions in the [discussion thread](https://osu.ppy.sh/community/forums/topics/719936) to help shape the future and direction of osu!mania mapping!
+
+### New Quality Assurance Team additions
+
+This month, we welcome a new piano masher to the Quality Assurance Team. Formerly a long standing member of the Beatmap Nominators, [Maxus](https://osu.ppy.sh/users/4335785) will be assisting the Quality Assurance Helpers with checking and disqualifying beatmaps, and giving [Feerum](https://osu.ppy.sh/users/4815717) a I a helping hand on the osu!mania community side of things. We're glad to have him on board!
```

Comment:
giving [Feerum](https://osu.ppy.sh/users/4815717) a I --> giving [Feerum](https://osu.ppy.sh/users/4815717) and me

Reply:
I suppose this is true since "Feerum and (x)" are the objects of the sentence, but I will forever hate how "(y) and me" looks, so I'll try for "Feerum and myself"